### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by andismith**
Fixed missing http which was leading to a 404

### CODE REVIEW

**Comment by Garbee** [comment]
lgtm once the build completes.

File: docs/_pages/faq.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ include_prefix: ../
 
 <h2 id="where-should-i-use">Where should I use Material Design Lite (MDL)?</h2>
 
-If you’re interested in a [Material Design](www.google.com/design/spec/material-design/introduction.html) experience using vanilla Web technologies like CSS, JavaScript and HTML, MDL might be a useful option to consider. We optimise for websites heavy on content, such as marketing pages, articles, blogs and general web content that isn’t particularly app-y.  If you just want to pick some colors, customise a template and ship a Material experience, we try to help make that process simpler.
+If you’re interested in a [Material Design](http://www.google.com/design/spec/material-design/introduction.html) experience using vanilla Web technologies like CSS, JavaScript and HTML, MDL might be a useful option to consider. We optimise for websites heavy on content, such as marketing pages, articles, blogs and general web content that isn’t particularly app-y.  If you just want to pick some colors, customise a template and ship a Material experience, we try to help make that process simpler.
```
Comment (surma) [inline docs/_pages/faq.md]: Would you mind turning that into `https://`?

Comment (Garbee) [inline docs/_pages/faq.md]: Oh right.

#### Start troll

Why isn't Google redirecting that on its own again?

#### end troll

**Comment by andismith** [comment]
Changed to https

**Comment by surma** [comment]
Thank you, @andismith! LGTM :)

**Pull request merged**